From 4473f5cb923ad4b9cd9e5a66e2205bec817af4fb Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Tue, 17 Jan 2023 18:35:57 -0800
Subject: [PATCH] Fix debug build

---
 core/deps/xbrz/xbrz.cpp | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/core/deps/xbrz/xbrz.cpp b/core/deps/xbrz/xbrz.cpp
index 9786e2d7..151a69ed 100644
--- a/core/deps/xbrz/xbrz.cpp
+++ b/core/deps/xbrz/xbrz.cpp
@@ -358,13 +358,6 @@ template <> inline unsigned char rotateBlendInfo<ROT_180>(unsigned char b) { ret
 template <> inline unsigned char rotateBlendInfo<ROT_270>(unsigned char b) { return ((b << 6) | (b >> 2)) & 0xff; }
 
 
-#ifndef NDEBUG
-    int debugPixelX = -1;
-    int debugPixelY = 12;
-    __declspec(thread) bool breakIntoDebugger = false;
-#endif
-
-
 /*
 input kernel area naming convention:
 -------------
@@ -392,11 +385,6 @@ void blendPixel(const Kernel_3x3& ker,
 #define h get_h<rotDeg>(ker)
 #define i get_i<rotDeg>(ker)
 
-#ifndef NDEBUG
-    if (breakIntoDebugger)
-        __debugbreak(); //__asm int 3;
-#endif
-
     const unsigned char blend = rotateBlendInfo<rotDeg>(blendInfo);
 
     if (getBottomR(blend) >= BLEND_NORMAL)
@@ -550,9 +538,6 @@ void scaleImage(const uint32_t* src, uint32_t* trg, int srcWidth, int srcHeight,
 
         for (int x = 0; x < srcWidth; ++x, out += Scaler::scale)
         {
-#ifndef NDEBUG
-            breakIntoDebugger = debugPixelX == x && debugPixelY == y;
-#endif
             //all those bounds checks have only insignificant impact on performance!
             const int x_m1 = std::max(x - 1, 0); //perf: prefer array indexing to additional pointers!
             const int x_p1 = std::min(x + 1, srcWidth - 1);
-- 
2.34.1

